.github/workflows/docs.yml

```diff
@@ -1,6 +1,8 @@
 name: Generate Documentation
 
-on: push
+on:
+  - push
+  - pull_request
```

Comment:
I think the pull_request event triggers for a bunch of different reasons, and we might want to be more specific on which ones we want to run CI on https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request has a list of activity types (including opened, closed, and review_requested) with some examples of when to use which types.

I'm guessing because we want it to only run when the pull request is merged, you want
```
types:
  - closed
```